force-app/main/default/internal-components/lwc/crmMessagingThreadViewer/newDesignTemplate.html

```diff
@@ -0,0 +1,179 @@
+                                    </template>
+                                </lightning-layout>
+                            </lightning-layout-item>
+                        </lightning-layout>
+                    </div>
+                    <template lwc:if={showspinner}>
+                        <div class="spinnerholder">
+                            <lightning-spinner alternative-text="Loading" size="medium"></lightning-spinner>
+                        </div>
+                    </template>
+                    <c-crm-messaging-quick-text
+                        conversation-note={textTemplate}
+                        reset-text-template={textTemplate}
+                        onlocklang={lockLangBtn}
+                    >
+                    </c-crm-messaging-quick-text>
+                </lightning-layout-item>
+                <lightning-layout-item size="12">
+                    <lightning-layout class="slds-var-p-top_medium slds-size_12-of-12">
+                        <!-- End dialogue section -->
+                        <lightning-layout-item class="slds-size_3-of-12">
+                            <template lwc:if={showClose}>
+                                <lightning-record-edit-form record-id={threadId} object-api-name="Thread__c">
+                                    <lightning-button
+                                        type="submit"
+                                        variant="destructive"
+                                        disabled={closedThread}
+                                        onclick={openModal}
+                                        label={labels.endDialogue}
+                                        class="slds-var-p-vertical_x-small endDialogBtn"
+                                    >
+                                    </lightning-button>
+                                </lightning-record-edit-form>
+                            </template>
+                        </lightning-layout-item>
+                        <!--Button container-->
+                        <lightning-layout-item class="slds-size_9-of-12">
+                            <lightning-button
+                                class="slds-float_right"
+                                type="submit"
+                                name="submit"
+                                label={submitButtonLabel}
+                                disabled={closedThread}
+                                variant="brand"
+                            >
+                            </lightning-button>
+                            <slot name="buttonContainer"></slot>
+                        </lightning-layout-item>
+                    </lightning-layout>
+                </lightning-layout-item>
+                <lightning-layout-item class="slds-size_12-of-12 slds-var-p-top_medium">
+                    <!--Flow section-->
+                    <slot name="flowSection"></slot>
+                    <!--Modal-->
```

Comment:
skrive om til å ikke være en modal? Heller bruk f.eks expandable. Inkludere CSS til render Se: https://developer.salesforce.com/docs/platform/lwc/guide/create-components-css.html#assign-css-stylesheets-to-a-component